docs/sources/getting-started/strategies.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Monitoring strategies"
+description = "Common monitoring strategies"
+keywords = ["grafana", "intro", "guide", "concepts", "methods"]
+type = "docs"
+[menu.docs]
+weight = 500
++++
+
+# Common observability strategies
+
+When you have a lot to monitor, like a server farm, you need a strategy to decide what is important enough to monitor. This page describes several common methods for choosing what to monitor.
+
+A logical strategy allows you to make uniform dashboards and scale your observability platform more easily.
+
+## Guidelines for usage
+
+- The USE method tells you how happy your machines are, the RED method tells you how happy your users are.
+- USE reports on causes of issues. 
+- RED reports on user experience and is more likely to report symptoms of problems.
+- The best practice of alerting is to alert on symptoms rather than causes, so alerting should be done on RED dashboards.
+
+## USE method
+
+USE stands for:
+
+- **Utilization -** Percent time the resource is busy, such as node CPU usage
+- **Saturation -** Amount of work a resource has to do, often queue length or node load
+- **Errors -** Count of error events
+
+This method is best for hardware resources in infrastructure, such as CPU, memory, and network devices.
```

Comment:
Add a link to http://www.brendangregg.com/usemethod.html?

Reply:
Done.